Update README.md (#130)
castrojo/awesome-immutable: README.md

		- Community & Downstream OCI Projects - These projects are built using Universal Blue infrastructure or base images and utilize BlueBuild to provide specialized versions of the uBlue stack
			- [WayBlue](https://github.com/wayblueorg/wayblue) - A community-driven collection of images providing lean, minimally-opinionated Wayland compositors (Hyprland, Sway, River, Niri, …) built on the uBlue framework
			- [SecureBlue](https://github.com/secureblue/secureblue) - A security-hardened project that adds kernel hardening, a hardened memory allocator (from GrapheneOS), and reduced attack surfaces to the Fedora Atomic/uBlue base

> 			- [RakuOS Linux](https://rakuos.org/) - A performance, gaming, and cutting-edge oriented Hybrid Atomic RPM-based bootable image featuring the KDE, GNOME, COSMIC, and NIRI desktop environments, using the [P03 kernel]([https://wiki.cachyos.org/features/kernel/](https://github.com/CatPieLeaf/linux-p03)), working with a native RPM package manager instead of `rpm-ostree`, and replacing legacy shell utilities with modern alternatives (i.e. `eza`, `bat`, `sudo-rs`, …)

Reviewer: ```suggestion
			- [RakuOS Linux](https://rakuos.org/) - A performance, gaming, and cutting-edge oriented Hybrid Atomic RPM-based bootable image featuring the KDE, GNOME, COSMIC, and Niri desktop environments, using the [P03 kernel](https://rakuos.org/kernel) by default, the [`Rum` package manager](https://rakuos.org/blog/introducing-rum), and replacing legacy shell utilities with modern alternatives (i.e. `eza`, `bat`, `sudo-rs`, …)
```

I didn't find a better link for the the `Rum` package manager than https://rakuos.org/blog/introducing-rum

Reviewer 2: Will make a dedicated page on the website soon and we can update you then :)
